modules/store.js

```diff
@@ -75,6 +75,15 @@ export const reducer = handleActions({
 
 }, initialState);
 
+export const reducer = function wrappedReducer(state, action) {
+  // if state is undefined then it can't be converted to mutable
+  let mutableState = state;
+  if (mutableState !== undefined) {
+    mutableState = getMutableState(state);
+  }
+  return getImmutableState(reduxAsyncReducer(mutableState, action));
+};
+
```

Comment:
Sorry I am a bit confused. Are you saying you want to export 2 reducers? So one export would be `reducer` and it would just be the regular reducer. And the second export would be the wrapped reducer as the immutable data one?